src/big/big_int.cr

```diff
@@ -201,6 +201,14 @@ struct BigInt < Int
     BigInt.new { |mpz| LibGMP.pow_ui(mpz, self, other) }
   end
 
+  def gcd(other : Int) : Int
+    BigInt.new { |mpz| LibGMP.gcd(mpz, self, other.to_big_i) }
```

Comment:
We should probably use `gcd_ui` here, probably checking that `other` is not negative (I don't know what should be the result then). The same should be done for `lcm(other : Int)`